src/features/chat/ui/CreateChatRoom.tsx

```
@@ -1,38 +1,25 @@
+      }}
+    >
       <input
-        onChange={(e: ChangeEvent<HTMLInputElement>) => setChatRoomTitle(e.target.value)}
-        value={chatRoomTitle}
+        onChange={(e: ChangeEvent<HTMLInputElement>) => handleChangeTitle(e)}
+        value={roomTitle}
+        placeholder="채팅방 생성"
+        className="border"
       />
-      <button>채팅 생성버튼</button>
+      <button>채팅방 생성</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)